### PR TITLE
Fix abyssal tentacle wasting whips on create with onCreate() charges

### PR DESCRIPTION
**Pull request by themrrobert**
### Description:

When Abyssal tentacle was launched as a degradeable, because the 10k charges were the default, and there was no way to add charges with the \/create method, we limited the user's ability to create multiple tentacles, and we DONT revert it to a Kraken tentacle. 

This was changed and has caused several people to lose their whips when attempting to re-create their abyssal tentacle.

### Changes:

- Adds onCreate() optional function to Creatable interface
- Inserts code into create.ts command to call and process the results of onCreate()
- Adds the onCreate() hook to the Abyssal tentacle creatable

### Other checks:

-   [x] I have tested all my changes thoroughly.

### CODE REVIEW

**Comment by Lajnux** [comment]
I would say the best short term fix would be to add #4044 instead. If a more centralised function is wanted after that can be resolved.

File: src/lib/data/creatables/slayer.ts [hunk 1]
```diff
@@ -130,7 +132,16 @@ export const slayerCreatables: Createable[] = [
 		}),
 		outputItems: new Bank({ 'Abyssal tentacle': 1 }),
 		GPCost: 0,
-		maxCanOwn: 1
+		maxCanOwn: 1,
```
Comment (Lajnux) [inline src/lib/data/creatables/slayer.ts:135]:
```suggestion
		maxCanOwn: 1,
		
```
Can be removed with this PR right?

Comment (themrrobert) [reply src/lib/data/creatables/slayer.ts:135]: It technically could be removed, however then you get into a situation where there's multiple Abyssal tentacles, all sharing the same charge pool, which I don't think is the best idea considering they degrade into just the tentacle.

Comment (Lajnux) [reply src/lib/data/creatables/slayer.ts:135]: Yea I see what you mean. Then it looks good. 🥰🐳

**Pull request merged**